pkg/platform/runtime/target/target.go

```diff
@@ -35,16 +36,34 @@ const (
 )
 
 // usageTriggers are triggers that indicate actual usage of the runtime (as oppose to simply making changes to the runtime)
-var usageTriggers = []Trigger{TriggerActivate, TriggerScript, TriggerDeploy, TriggerExec}
+var usageTriggers = []Trigger{
+	TriggerActivate,
+	TriggerScript,
+	TriggerDeploy,
+	TriggerExec,
+	TriggerBranch,
+	TriggerImport,
+	TriggerPackage,
+	TriggerPull,
+	TriggerReset,
+	TriggerRevert,
+}
```

Comment:
I think at this point we should probably invert this to a blacklist rather than a whitelist :p Will file a followup.